pkg/features/kube_features.go

```diff
@@ -168,7 +168,7 @@ const (
 	// deprecated: v1.10
 	//
 	// Enable the service proxy to contact external IP addresses. Note this feature is present
-	// only for backward compatability, it will be removed in the 1.11 release.
+	// only for backward compatibility, it will be removed in the 1.11 release.
```

Comment:
This has been fixed in https://github.com/kubernetes/kubernetes/pull/59464 - https://github.com/kubernetes/kubernetes/pull/59464/files#diff-84e72664a7cc9add01954d3a7b8cbb45. Closing.

/close